Src/Base/AMReX.cpp

```diff
@@ -51,7 +51,7 @@
 #include <omp.h>
 #endif
 
-#if defined(__APPLE__)
+#if defined(__APPLE__) && !defined(__arm64__)
```

Comment:
Rather than `!defined(__arm64__)`, these should probably be rewritten in the affirmative, for the x86_64 target that does support SSE intrinsics:

```c++
#if defined(__APPLE__) && defined(__x86_64__)
```

Reply:
Good point.  @ax3l Could you add this to your PR #2672?

Reply:
ok will do